tests/TransformationTests.elm

```diff
@@ -0,0 +1,104 @@
+module TransformationTests exposing (suite)
+
+import Expect
+import Fuzz exposing (floatRange, int, intRange, string)
+import Test exposing (..)
+import Transformation exposing (toModel, withField, withoutField)
+import Validation exposing (Field, preValidatedField)
+
+
+suite : Test
+suite =
+    describe "The Transformation module"
+        [ describe "transforms form to model with correct inputs"
+            [ fuzz3 (intRange 1 100) (floatRange 0.1 99.9) string "should return 'Ok model'" <|
```

Comment:
Nemam namisto fce `fuzz3` pouzit `Fuzzer` na `Form`?

Reply:
Klidne bych to nechal takto.